src/frontend/api.js

```diff
@@ -137,6 +142,19 @@ export function Api({
 
   nodejs.channel.addListener("status", onStatus);
 
+  nodejs.channel.addListener("copy-apk", async () => {
+    const tmpDir = RNFS.DocumentDirectoryPath + "/installer";
+    log("+++ 5", tmpDir);
+    const apkName = "mapeo.apk";
+    const tmpApkPath = `${tmpDir}/${apkName}`;
+    await RNFS.mkdir(tmpDir);
+    log("+++ 6");
+    await RNFS.copyFile(AppInfo.sourceDir, tmpApkPath);
+    log("+++ 7");
+    // await RNFS.unlink(tmpDir);
+    nodejs.channel.post("apk-ready", tmpApkPath, APP_VERSION);
+  });
+
```

Comment:
Why do the file copy in front-end code rather than nodejs-mobile? This feels like it belongs in the backend.